pyteal/ast/binaryexpr_test.py

```diff
@@ -1,1499 +1,1560 @@
 import pytest
 
-from .. import *
+import pyteal as pt
```

Comment:
Yes, I haven't been able to figure out how to take the top level relative import `.` into an alias `pt`. The [example I gave in the previous discussion](https://github.com/algorand/pyteal/pull/273#discussion_r849748369) also imports this way (`from pyteal import pt`). I probably should have been more explicit in stating that this was my plan. 

There is one advantage to this approach. We are asserting a non-relative import path in the unit test, and often explicit is preferred to implicit in unit tests. Additionally, this eases the way to moving all the unit tests into `tests/unit` if that is something we choose to do in the future.

Reply:
I see the issue, `import .. as pt` does not work since it requires an absolute module path, and I don't see a way around that.

I'll think about the ramifications of this, but it does not block this PR in any way.